crud.lua

```diff
@@ -91,4 +85,13 @@ function crud.init()
     select.init()
 end
 
+function crud.init_router()
+   rawset(_G, 'crud', crud)
+end
+
+function crud.init()
```

Comment:
I don't like this function. Nobody takes care about `init_router` and `init_storage` when `init` function exists.